### DISCUSSION TITLE
Disable the `esc` in showModal

### DISCUSSION THREAD

**Discussion by gs** [saadeghi/daisyui#2075] [General]
Currently by hitting `esc` the modal can be closed. 
However, I would like to disable this functionality, so user needs to press the button. 

Is it  possible to pass a flag to `showModal` to make the modal not dismissible by hitting `esc`?

**Comment by saadeghi** [comment]
From Stack Overflow: https://stackoverflow.com/a/61021326  

>You can try to use cancel event
```
dialog.addEventListener('cancel', (event) => {
    event.preventDefault();
});
```

**Reply by gs** [reply]
Thank you! That worked prefectly!
